<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>primary</class>
 <widget class="QDialog" name="primary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Primary Parameters</string>
  </property>
  <widget class="QPushButton" name="primary_ok">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>660</y>
     <width>96</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="primary_back">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>660</y>
     <width>96</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>151</width>
     <height>248</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>MATERIAL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="aluminium_rb">
      <property name="text">
       <string>Aluminium</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="steel_rb">
      <property name="text">
       <string>Steel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="castiron_rb">
      <property name="text">
       <string>Cast Iron</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="other_material_rb">
      <property name="text">
       <string>Other</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mat_rho_layout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="youngs_modulus_label">
          <property name="text">
           <string>E</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mat_rho_label">
          <property name="text">
           <string>Rho</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="youngs_mod_le">
          <property name="text">
           <string>200</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mat_rho_le">
          <property name="text">
           <string>7000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="unit_E_label">
          <property name="text">
           <string>GPa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="unit_mat_rho_label">
          <property name="text">
           <string>Kg/m3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="Material_select_pb">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>600</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Length</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="beam_length_le">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>m</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>135</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>CROSS SECTIONS</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="solidCircle_rb">
        <property name="text">
         <string>Solid Circle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="hollowCircle_rb">
        <property name="text">
         <string>Hollow Circle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rectangle_rb">
        <property name="text">
         <string>Rectangle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="square_rb">
        <property name="text">
         <string>Square</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="iSection_rb">
        <property name="text">
         <string>I - Section</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sectionSelect_pb">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="csa_ln">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="moi_ln">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>154</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Cross-sectional Area</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>134</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Moment of Inertia</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>123</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Material Density</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>124</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Youngs modulus</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="E_ln">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="rho_ln">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
